### PR TITLE
SPOI-10928 Example application for Kinesis Input Operator.

### PR DESCRIPTION
**Pull request by deepak-narkhede**

### CODE REVIEW

**Comment by deepak-narkhede** [comment]
@shubham-pathak22  or @yogidevendra. Please review.

**Comment by shubham-pathak22** [comment]
Please add a Readme.md file. You may refer to existing examples.
https://github.com/DataTorrent/examples/tree/master/tutorials/jmsActiveMQ

File: tutorials/kinesisInput/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+| Property Name  | Description |
+| -------------  | ----------- |
+| dt.operator.kinesisInput.prop.streamName | AWS Kinesis Stream name |
+| dt.operator.kinesisInput.prop.accessKey | AWS Credentials AccessKeyId  |
+| dt.operator.kinesisInput.prop.secretKey | AWS Credentials SecretAccessKey |
+| dt.operator.kinesisInput.prop.endPoint | AWS EndPoint  |
+| dt.operator.fileOutput.prop.outputFileName | Output file name e.g: output.txt |
+| dt.operator.fileOutput.prop.filePath | HDFS output file path |
+
```
Comment (shubham-pathak22) [inline tutorials/kinesisInput/README.md]: Could you add description for initialOffset property?

Comment (deepak-narkhede) [inline tutorials/kinesisInput/README.md]: Thanks. Done.

**Pull request merged**